### DISCUSSION TITLE
Introducing the NetBox plugin ideas board

### DISCUSSION THREAD

**Discussion by jeremystretch** [netbox-community/netbox#12689] [Announcements]
NetBox first introduced support for plugins way back in version 2.8, and the [plugins framework](https://docs.netbox.dev/en/stable/plugins/development/) has grown substantially since then to enable users to extend the application with their own models and business logic. Today there are dozens of [popular open source plugins](https://netbox.dev/plugins/) available, and many times more in private development. Plugins provide an ideal avenue for the implementation of new features that aren't suitable for the core application, but until now we haven't had a great solution for tracking proposed plugins.

On [today's community call](https://go.netboxlabs.com/may-netbox-community-meetup), we announced a new forum dedicated exclusively to proposing and discussing ideas for new plugins: the **[NetBox plugin ideas board](https://plugin-ideas.netbox.dev/)**. Here, anyone can submit a new plugin idea, and vote on and discuss ideas submitted by others.  The relative popularity of each idea can be gauged by the number of votes it receives, and interested parties can collaborate to turn rough ideas into executable strategies.

If you've ever had an idea for a plugin, this is the place to share it! I look forward to seeing what new ideas pop up here.

**Comment by dreng** [comment]
Hi Jeremy,

Although I like the idea very much, I would not like to register for yet another service to participate in discussions and cast votes there. Wouldn't it be possible to authenticate via Github?
